Makefile

```diff
@@ -6,4 +6,14 @@ changelog:
 release:
 	semtag final -s minor
 
+.PHONY: test
 test:
+	pre-commit run
+
+.PHONY: test-slow
```

Comment:
This an the other PHONY below need updated target names

Reply:
thanks for catching, updated